flink-clients/src/main/java/org/apache/flink/client/program/rest/RestClusterClient.java

```diff
@@ -359,7 +333,8 @@ public CompletableFuture<JobID> submitJob(@Nonnull JobGraph jobGraph) {
                         () -> {
                             try {
                                 final java.nio.file.Path jobGraphFile =
-                                        Files.createTempFile(tempDir, "flink-jobgraph", ".bin");
+                                        Files.createTempFile(
+                                                "flink-jobgraph-" + jobGraph.getJobID(), ".bin");
```

Comment:
I have to add the jobid into the job graph file name, in order to check whether the job graph file has been deleted in `testJobGraphFileCleanedUpOnJobSubmissionFailure`. cc @zhuzhurk